functions.php

```diff
@@ -112,6 +112,27 @@ function insert_into_template(positioning_text, added_text, insert_before) {
 			wp_add_inline_script( 'jquery', $script );
 		}
 	}
+
+	if ( 'themes.php' === $hook || 'plugins.php' === $hook ) {
+		wp_enqueue_script(
+			'wp-autoupdates',
+			plugin_dir_url( __FILE__ ) . 'js/wp-autoupdates.js',
+			array( 'jquery', 'wp-ajax-response' ),
+			'1.0.0',
```

Comment:
Can you replace `'1.0.0'` with the new `WP_AUTO_UPDATES_VERSION` constant?

Reply:
@pbiron done with https://github.com/WordPress/wp-autoupdates/pull/61/commits/7da442f339e7fbcdb28d10031d940a6c04e13e62